MuvrKit/MKMarkovPredictor+JSON.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public extension MKMarkovPredictor {
+    
+    public var json: NSData {
+        var planJson: [String: [String : AnyObject]] = [:]
+        weightPlan.forEach {(exerciseId, weightPlan) in
+            planJson[exerciseId] = weightPlan.metadata {"\($0)"}
+        }
+        NSLog("\n\n\n\njson = \(planJson)")
+        return try! NSJSONSerialization.dataWithJSONObject(["weightPlan": planJson, "simpleScalars":simpleScalars], options: [])
+    }
+    
+    public func mergeJSON(data: NSData) {
```

Comment:
This method's usage has changed. I think it will have to be removed, and we'll need to have (convenience) initializer in each of the implementations of the protocol. That is,

```swift
class MKPoly...Predictor {
    init?(json: AnyObject) {
        ///  inverse of ``.json``
    }
}
```
and
```swift
class MKMarkovPredictor : ... {
    init?(json: AnyObject) {
    }
}
```